### DISCUSSION TITLE
Duda sobre implementacion en "Juego de espias"

### DISCUSSION THREAD

**Discussion by JeanFranco99** [INGCOM-UNRN-P1/cursada-2023#238] [Preguntas y respuestas] [answered]
Buenas! Estaba terminando el tp integrador y me entro una duda.
`$> ./generador tabla` 
Cada vez que se la llame, va a generar una tabla aleatoria que solo se utiliza 
`$> ./substitucion c tabla entrada salida` 
No tiene ningun otro uso? 
Esta tabla se guarda en un .txt o como seria el funcionamiento correcto/adecuado?

**Comment by martinvilu** [comment, marked as answer]
Generador es una forma rápida de armar las tablas para el algoritmo de substitución, pero debiera de ser configurable, sobre qué caracteres trabajar, la forma de mezclar, etc.

No solo decir a donde, sino de que forma tambien.

**Reply by JeanFranco99** [reply]
En la forma de mezclar aplique el algoritmo de fisher yates que es el que se usa para juegos de barajar al azar. Sobre que caracteres, podría ser dentro de un conjunto o rango?

**Reply by martinvilu** [reply]
Interesante lo de la mezcla.

¿Por qué no ambas formas de indicar?
